ampd/src/handlers/config.rs

```diff
@@ -21,22 +23,26 @@ pub enum Config {
         cosmwasm_contract: TMAddress,
         #[serde(flatten, with = "chain")]
         chain: Chain,
+        rpc_timeout: Option<Duration>,
```

Comment:
does it `#[serde(with = "humantime_serde")]` work with opton? can do `5s` with this

Reply:
Can try